data/json/terrain.json

```diff
@@ -348,6 +365,7 @@
         "type" : "terrain",
         "id" : "t_dirtmound",
         "name": "mound of dirt",
+        "description": "An area of heaped dirt, not easily traversible but if examined more closely, they could be quite favorable for planting seeds and the like.",
```

Comment:
Hm, this one's a little confusing. Perhaps something like:

> An area of heaped dirt blocking movement. Could be quite favorable for planting seeds.

Reply:
This dirt mount doesn't block, it just has increased move cost, hence the "not easily traversible" as in, difficult to walk across.